scss/footer.scss

```diff
@@ -0,0 +1,19 @@
+.train-footer{
+  background-color: $training-blue; 
+  display: flex;
+  flex-direction: column; 
+  padding: 40px 20px 20px 20px;  
+
+  .txt {
+  	color: $training-white;
```

Comment:
this properties should be indented using 2 spaces instead of 1 tab (or 4 spaces)